examples/website/textures/components/compressed-texture.js

```diff
@@ -228,6 +218,39 @@ export default class CompressedTexture extends PureComponent {
     return canvas.toDataURL();
   }
 
+  async getLoaderData(image) {
+    let loader = null;
+    let arrayBuffer = null;
+    let length = 0;
+    let src = '';
+
+    // eslint-disable-next-line no-undef
+    if (image instanceof File) {
+      arrayBuffer = await image.arrayBuffer();
+      length = image.size;
+      loader = await selectLoader(arrayBuffer, [
```

Comment:
`selectLoader`, `parse` etc accept file objects directly. No need to check separately.

Reply:
done